extensions/amp-social-share/0.1/amp-social-share.css

```diff
@@ -70,3 +70,7 @@ amp-social-share {
 .amp-social-share-whatsapp {
   background-color: #25d366;
 }
+
+.amp-social-share-system {
+  background-color: deeppink;
```

Comment:
Changed to white on black. (Was originally deeppink to make it clear it hadn't been "designed"!)